test/metric_value_benchmark/eks_daemonset_test.go

```diff
@@ -66,19 +77,80 @@ func (e *EKSDaemonTestRunner) validateInstanceMetrics(name string) status.TestRe
 	return testResult
 }
 
-func (e *EKSDaemonTestRunner) GetTestName() string {
+func (e *EKSDaemonTestRunner) validateLogs(eks *environment.MetaData) status.TestResult {
+	testResult := status.TestResult{
+		Name:   "emf-logs",
+		Status: status.FAILED,
+	}
+
+	now := time.Now()
+	group := fmt.Sprintf("/aws/containerinsights/%s/performance", eks.EKSClusterName)
+
+	// need to get the instances used for the EKS cluster
+	EKSInstances, err := awsservice.GetEKSInstances(eks.EKSClusterName)
+	if err != nil {
+		log.Println("failed to get EKS instances")
+		return testResult
+	}
+
+	for _, instance := range EKSInstances {
+		validateLogContents := func(s string) bool {
+			return strings.Contains(s, fmt.Sprintf("\"ClusterName\":\"%s\"", eks.EKSClusterName))
+		}
+
+		var ok bool
+		stream := *instance.InstanceName
+		ok, err = awsservice.ValidateLogs(group, stream, nil, &now, func(logs []string) bool {
```

Comment:
I think that the param is in the wrong spot here. 
```
func ValidateLogs(logGroup, logStream string, since, until *time.Time, validator func(logs []string) bool) (bool, error) {
```

The third param should probably be `&now` and the fourth param should be `nil`

Reply:
🤔 I guess it depends on when `now` is defined relative to the logs being generated... if that's what it is right now and the ECS CI tests are working, then I'll live with it